docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -0,0 +1,119 @@
+## Deploy a Nutanix Cluster
+
+Use the following steps to deploy a Kubernetes cluster in Nutanix.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** select **Clusters**.
+
+3. Click on **Add New Cluster** and select **Deploy New Cluster** on the next page that Palette displays. 
+
+4. Select **Nutanix** and click the **Start Nutanix Configuration** button.
+
+5. Fill out the following basic information, and click **Next** to continue.
+
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Cluster Name**| A custom name for the cluster. |
+  | **Description**| Use the description to provide context about the cluster.|
+  | **Tags**| Assign any desired cluster tags. Tags on a cluster are propagated to the Virtual Machines (VMs) deployed to the target environments.|
+  | **Cloud Account** | Select your Nutanix account from the **drop-down Menu**. |
+
+6. Select the Nutanix cluster profile you created and click **Next**. Palette displays the profile layers.
+
+7. Review profile layers, leaving the OS and Kubernetes packs empty, and customize parameters as desired in the YAML files that display when you select the network and storage layers. Click **Next** when you are done.
+
+8. In the Cluster configuration YAML file that Palette displays, edit the file to replace variables within curly braces listed in the table below with values that apply to your Nutanix cloud environment, and make any adjustments to configure your cluster. Click **Next** when you are done.
+
+  | **Variable** | **Description** |
+  |--------------|-----------------|
+  | `${CLUSTER_NAME}`| The name of the Nutanix workload cluster. |
+  | `${CONTROL_PLANE_ENDPOINT_IP}`| The Nutanix Prism Central IP address. |
+  | `${NUTANIX_ENDPOINT}`| The Nutanix cluster IP address. |  
+
+  :::caution
+
+  The following applies when replacing variables within curly braces in the YAML configuration files.
+
+    - All the variables must be resolved or have a default value.
+
+    - Verify default values such as the port.
+
+    - Names you provide must match. Any names in the YAML files that do not match your Nutanix cluster configuration will result in unsuccessful cluster deployment.
+
+    - Values that are passed as a string, such as names and keys, must be enclosed in quotes, for example `" "`.
+
+    - When replacing values, remove the dollar sign and curly braces.
+
+  
+  :::
+
+9. In the Node pool configuration YAML files for the master and worker pools, edit the files to replace each occurrence of the variables within curly braces listed in the tables below with values that apply to your Nutanix cloud environment. You can configure scaling in the Palette UI by specifying the number of nodes in the pool. This corresponds to `replicas` in the YAML file.
+
+  #### Master Pool 
+
+  | **Variable** | **Description** |
+  |--------------|-----------------|
+  | `${CLUSTER_NAME}`| The name of the Nutanix workload cluster. |
```

Comment:
Please add a comment that it must be the exact same name as the one created in palette in step 5

Reply:
Revised the description as: The name of the Nutanix workload cluster. Use the same cluster name you specified in step 5.